libs/designer/src/lib/ui/connections/dropzone.tsx

```diff
@@ -360,6 +363,13 @@ export const DropZone: React.FC<DropZoneProps> = memo(({ graphId, parentId, chil
                   </MenuItem>
                 </>
               )}
+              {UiInteractionsService()?.getAddButtonDropdownMenuItems?.({graphId, parentId, childId, preloadedOperations, dispatch}).map((item, index) => {
```

Comment:
please add a test here to make sure this still works when there is no service. That could be a regression point later